core/selection.js

```diff
@@ -1168,7 +1168,7 @@
 					range = sel.createRange();
 
 				// The selection range is reported on host, but actually it should applies to the content doc.
-				if ( sel.type != 'None' && range.parentElement().ownerDocument == doc.$ )
+				if ( sel.type != 'None' && ( range.parentElement() || {}).ownerDocument == doc.$ )
```

Comment:
Ok so i change it also a little bit:
```
if ( sel.type != 'None' && range.parentElement() && range.parentElement().ownerDocument == doc.$ )
```
So if 2nd part of condition failed then 3rd one won't be evaluated.